contracts/axelar-gas-service/tests/test.rs

```diff
@@ -26,20 +26,19 @@ fn setup_env<'a>() -> (Env, Address, Address, AxelarGasServiceClient<'a>) {
     (env, contract_id, gas_collector, client)
 }
 
-fn setup_gas_token(env: &Env, gas_amount: i128) -> Token {
+fn mint_gas_token(env: &Env, recipient: &Address, gas_amount: i128, mint_amount: i128) -> Token {
```

Comment:
```suggestion
fn setup_token(env: &Env, recipient: &Address, amount: i128) -> Token {
```
make this generic so it can be used for any token. For specific tests where the amount provided as input is different than minted amount, construct the Token struct manually in the test. Otherwise this method is confusing for the normal usage, and especially as a testutil

Reply:
make sense. updated.